src/analysis/locally_convex/with_seminorms.lean

```diff
@@ -289,6 +289,82 @@ begin
   rw (congr_fun (congr_arg (@nhds E) hp.1) 0),
   exact add_group_filter_basis.nhds_zero_has_basis _,
 end
+
+/-- The `x`-neighbourhoods of a space whose topology is induced by a family of seminorms
+are exactly the sets which contain seminorm balls around `x`.-/
+lemma with_seminorms.mem_nhds_iff (hp : with_seminorms p) (x : E) (U : set E):
+  U ∈ nhds x ↔ ∃ (i : finset ι) (ε > 0), (i.sup p).ball x ε ⊆ U :=
+begin
+  rw [hp.with_seminorms_eq, (p.add_group_filter_basis.nhds_has_basis x).mem_iff' U],
+  split,
+  { rintros ⟨V, V_in_basis, V_sub_U⟩,
+    rcases p.basis_sets_iff.mp V_in_basis with ⟨i_set, ε, ε_pos, hr⟩,
```

Comment:
```suggestion
    rcases p.basis_sets_iff.mp V_in_basis with ⟨s, ε, ε_pos, hV⟩,
```
I don't see a reason why this variable should be called `hr`.

Generally, we tend to use `h$foo` for hypotheses on `$foo`, and your `V_sub_U` for instance might become `hVU`. I don't have a strong opinion here, but maybe someone else has.

Reply:
The ```hr``` was a slip-up, thank you! I tried to make the variable names a bit more consistent. I did not immediately find any authoritative source on the comments beyond [the style guide](https://leanprover-community.github.io/contribute/style.html), so I just tried to go with a mix of the ```hVU``` and ```V_subset_U``` styles that maximizes readability in my own subjective sense.